stixcore/idb/manager.py

```diff
@@ -139,14 +188,14 @@ def download_version(self, version_label, force=False,
             try:
                 Path(self._get_filename_for_version(version_label)).unlink()
             except Exception as e:
-                logger.warn(e)
+                logger.warning(e)
 
         vlabel = (IDB_VERSION_PREFIX + IDBManager.convert_version_label(version_label))
         vdir = self.data_root / vlabel
         try:
             if not vdir.exists():
```

Comment:
`vdir.mkdir( parents=True, exist_ok=True)` will create if doesn't exit and do nothing if it does